packages/babel-helpers/src/index.js

```diff
@@ -20,14 +20,32 @@ function makePath(path) {
 function getHelperMetadata(file) {
   const globals = new Set();
   const localBindingNames = new Set();
+  // Maps imported identifier -> helper name
+  const dependencies = Object.create(null);
 
   let exportName;
   let exportPath;
   const exportBindingAssignments = [];
+  const importPaths = [];
+  const importBindingsReferences = [];
 
   traverse(file, {
     ImportDeclaration(child) {
-      throw child.buildCodeFrameError("Helpers may import anything.");
+      const name = child.node.source.value;
+      if (!helpers[name]) {
+        throw child.buildCodeFrameError(`Unknown helper ${name}`);
```

Comment:
This actually throws `Cannot read property 'file' of undefined`, but it is not directly related to this PR (also the others `throw child.buildCodeFrameError` in this file throw that error). I will fix it separately.